api/NodeList.json

```diff
@@ -181,7 +181,7 @@
               "version_added": "38"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": true
```

Comment:
Based on our existing data (which maps Opera releases to Blink), I'd wager that this is 41:

```suggestion
              "version_added": "41"
```